docs/content/en/docs-dev/quickstart/_index.md

```diff
@@ -8,135 +8,109 @@ description: >
 
 This page is a guideline for installing PipeCD into your Kubernetes cluster and deploying a "hello world" application to that same Kubernetes cluster.
 
-Note: It's not required to install the PipeCD control plane to the cluster where your applications are running. Please read this [blog post](/blog/2021/12/29/pipecd-best-practice-01-operate-your-own-pipecd-cluster/) to understand more about PipeCD in real life use cases.
+Note:
 
-Note: If you want to experiment with PipeCD freely or don't have a Kubernetes cluster, we recommend [this Tutorial](https://github.com/pipe-cd/tutorial).
+- It's not required to install the PipeCD control plane to the cluster where your applications are running. Please read this [blog post](/blog/2021/12/29/pipecd-best-practice-01-operate-your-own-pipecd-cluster/) to understand more about PipeCD in real life use cases.
+- If you want to experiment with PipeCD freely or don't have a Kubernetes cluster, we recommend [this Tutorial](https://github.com/pipe-cd/tutorial).
 
 ### Prerequisites
 - Having a Kubernetes cluster and connect to it via [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
-- Forked the [Examples](https://github.com/pipe-cd/examples) repository
-- Prepare for authentication to [GitHub Container Registry](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#authenticating-to-the-container-registry).
 
 ### 1. Installing PipeCD in quickstart mode
 
-#### 1.1. Installing PipeCD client
+Across the [PipeCD concepts](/docs/concepts/), PipeCD platform is constructed by 2 components: Control Plane and Piped (the agent).
 
-##### Method 1: Official Installation
-The official PipeCD client named `pipectl` can be installed using the following command
-
-``` console
-# OS="darwin" or "linux"
-curl -Lo ./pipectl https://github.com/pipe-cd/pipecd/releases/download/{{< blocks/latest_version >}}/pipectl_{{< blocks/latest_version >}}_{OS}_amd64
-```
-
-Then make the pipectl binary executable
-
-``` console
-chmod +x ./pipectl
-```
-
-You can also move the pipectl binary to the $PATH for later use
+#### 1.1. Installing PipeCD Control Plane
 
 ```console
-sudo mv ./pipectl /usr/local/bin/pipectl
+$ kubectl create namespace pipecd
```

Comment:
[imo] It might be nice to add a section for the namespace. User can understand both of the components are installed in the namespace in advance.

Reply:
https://argo-cd.readthedocs.io/en/stable/getting_started/
I just use the same method here, I don't want to add too many code section, since we already have a lot of images in register piped section 👀

Reply:
OK, I got your point.